src/Component/State/CallableProcessor.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\State;
+
+use Psr\Container\ContainerInterface;
+use Sylius\Component\Resource\Context\Context;
+use Sylius\Component\Resource\Metadata\Operation;
+use Webmozart\Assert\Assert;
+
+/**
+ * @experimental
+ */
+final class CallableProcessor implements ProcessorInterface
```

Comment:
This class is not responsible for callable processors only
```suggestion
final class Processor implements ProcessorInterface
```
or 
```suggestion
final class ProcessorExecutor implements ProcessorInterface
```